src/Microsoft.Health.Fhir.Core/Features/Operations/Export/ExportJobTask.cs

```diff
@@ -83,7 +83,7 @@ public class ExportJobTask : IExportJobTask
             _resourceDeserializer = resourceDeserializer;
             _exportDestinationClient = exportDestinationClient;
             _anonymizerFactory = anonymizerFactory;
-            _mediator = mediator;
+            _mediator = mediatorFactory.Invoke();
```

Comment:
I think we should wait to invoke this until it is used. The factory is what should be stored here.

Reply:
Makes sense. I did this approach because I was worried the scope would be disposed already haha. But we can adjust later if needed.

Reply:
fixed, thanks RB 👍🏽